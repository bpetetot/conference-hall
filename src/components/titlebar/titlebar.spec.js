/* eslint-env jest */
import snap from 'tests/snapshot'
import Titlebar from './index.js'

jest.mock('../iconLabel', () => 'IconLabel')

const snapshot = props => snap(Titlebar)({ ...props })

describe('components/titlebar', () => {
  it(
    'should render',
    snapshot({
      icon: 'i',
      title: 't',
      children: 'c',
      className: 'cn',
    }),
  )
})
