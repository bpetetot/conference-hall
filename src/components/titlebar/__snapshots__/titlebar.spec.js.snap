// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/titlebar should render 1`] = `
<div
  className="titlebar cn"
>
  <span
    className="titlebar-title"
  >
    <IconLabel
      icon="i"
      label="t"
      overflow={true}
    />
  </span>
  <div
    className="titlebar-actions"
  >
    c
  </div>
</div>
`;
